docs/source/faq.rst

```diff
@@ -120,6 +120,42 @@ section of the SimpleITK Visual Studio/C# build instructions.
 How to Use
 ==========
 
+How many threads should I use when using SimpleITK and performing multiprocessing?
+----------------------------------------------------------------------------------
+
+By default the number of threads used by SimpleITK/ITK multi-threaded
```

Comment:
Please start off with the positive and how things are:

"SimpleITK by default multi-threads most image filters to utilize all virtual cores  available to the process." Then maybe describe thread pool vs os multi-threader.

Enumerate a these problematic cases, grouped by solution.

Reply:
Modified to start with a positive message, but I don't think we should go into discussions of the various threading models in the FAQ. 

Not sure where to discuss the differences between the thread-pool vs platform threads: former has a fixed size and won't be changed for the current process vs. later where the number of threads can shrink and grow more flexibly. On the other hand the platform threads take more time to spin up vs. the pool where the threads are spun up once and reused. 

Also, this is a compile time decision so not sure this matters to the user who is likely using a pre-compiled binary. Maybe add in the building SimpleITK section?